arches_controlled_lists/datatypes/datatypes.py

```diff
@@ -46,24 +48,19 @@ def to_python(self, value: Iterable[Mapping] | None) -> list[Reference] | None:
         return references or None
 
     def serialize(self, value):
+        """
+        Unlike to_json() which is concerned with calculating display values, this
+        method simply serializes all the information held in the Reference(s).
+        """
         if value is None:
             return None
         return [
             asdict(reference) if isinstance(reference, Reference) else {**reference}
             for reference in value
         ]
 
-    def to_representation(self, value):
-        references = self.to_python(value)
-        if not references:
-            return None
-        return [
-            {
-                "list_item_id": reference.labels[0].list_item_id,
-                "display_value": ListItem.find_best_label_from_set(reference.labels),
-            }
-            for reference in references
-        ]
+    def to_json(self, tile, node):
+        return {"@display_value": self.get_display_value(tile, node)}
```

Comment:
The base datatype and some other datatypes call `compile_json` which accepts kwargs in addition to tile & node. Should we instead be overwriting `compile_json`?

Reply:
Given the latest refactor I can just remove `to_json()` entirely. 👍